src/www/app/outline_server_repository/index.ts

```diff
@@ -78,8 +78,8 @@ export class OutlineServerRepository implements ServerRepository {
   }
 
   add(accessKey: string) {
-    const config = accessKeyToShadowsocksConfig(accessKey);
-    const server = this.createServer(uuidv4(), accessKey, config.name);
+    const config = accessKeyToServerConfig(accessKey);
```

Comment:
All we really need here is to extract the name from the key. You could call `SHADOWSOCKS_URI.parse` and get the tag.